proposed/2023/vs-filters.md

```diff
@@ -0,0 +1,89 @@
+# Visual Studio Filters & Sorting Options
+
+- [Jon Douglas](https://github.com/JonDouglas)
+- 2023-06-23
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+This proposal represents a common ask for more filters and sorting options in the NuGet Package Manager experience in Visual Studio. 
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+As NuGet continues to expand as a package ecosystem, there has become a dire need for being able to filter and sort packages through one's daily management in their IDE of choice. While NuGet.org supports a plethora of filters and sorts on the search experience today, we'd like to bring these similar experiences to the various tabs(browse, installed, etc) in Visual Studio to help aid various jobs-to-be-done when it comes down to package management.
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or pseudocode to show how this proposal would look. -->
+
+When a developer is using Visual Studio to search and manage NuGet packages, they will see a new filter and sort experience. 
+
+The browse filter experience will allow the developer to filter results based on common concepts such as "frameworks", "prerelease", "owners", "download count", "license", and "created on". 
+
+![](resources/VSFilter/PackageOptions.png)
+
+The installed filter experience will allow the developer to filter results based on all the **browse filter** and additional package status filters such as "deprecated", "vulnerable", and "update".
+
+![](resources/VSFilter/PackageOptionsAndStatus.png)
```

Comment:
If we're adding a has updates filter, I think we should consider removing the updates tab. 
IMO, it'd inconsistent to have a filter that basically makes the installed tab and updates tab. 

The *only* thing that the updates tab now has is that it has checkboxes that allow you to start the operation on multiple packages.

Reply:
yeah or we can just omit updates as a filter and bring the filtering/sort experiences to updates tab too.

Reply:
Fair. Given the amount of work that'd go into this, I'd love for us to consider removing the updates tab as a parallel workstream. 

It's one of those, if you do them one after the other, it might take 3 weeks each for a total of 6, but doing them together is probably just 4 weeks.

Reply:
May need to look again in the actual usage of update tab, but this makes sense to me.

Reply:
Updates tab could also be thought of as a shortcut to quickly filter. It's essentially enabling the "show updates" filter, without the option to disable it.

Reply:
Not part of this spec discussion, but it may be worthwhile to discuss the future of "Updates" tab - removing it or overloading it for quick "attention seeker" tab for all the 3 filters together?
![image](https://github.com/NuGet/Home/assets/14800916/fd5da05d-c7c8-42b1-a19c-2caaa33f3a2f)

Reply:
Btw, UI for package status seems like "enable" buttons and less of filters. I am assuming these will go through UX scrub.